lib/bundler/cli/add.rb

```diff
@@ -0,0 +1,42 @@
+# frozen_string_literal: true
+require "bundler/cli/common"
+require "net/http"
+require "yaml"
+
+module Bundler
+  class CLI::Add
```

Comment:
I'll hold off on more extensive review of this until `bundle add` and `bundle inject` are sharing similar functionality/code and internal interfaces.